.gitignore

```diff
@@ -0,0 +1,10 @@
+# Logs
+logs
+*.log
+
+# Dependency directories
+node_modules/
+bower_components/
+
+# Mac specific
+.DS_Store
```

Comment:
Hrmm... I have not added compiled files before to the repo. Am assuming that we'd run a `bower install` and `npm install` when the CI is running, if we do so desire. It would be good to check that those commands work properly rather than test the actual compiled files themselves.